hack/setup_env.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+set -o xtrace
+
+yum install -y docker libvirt-clients awscli python3-pip postgresql genisoimage && \
+    yum clean all
```

Comment:
I understand the intention of this script is mostly to be run in CI and the Docker images but thought I would point out this will fail for non-rhel OSes :) 

The only reason I'm bringing this up is because you added an `init` target to the `Makefile` :smile:

Reply:
You are right. I don't know if we want to support those ;-)